hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/AbstractTestS3AEncryption.java

```diff
@@ -79,7 +79,7 @@ protected void patchConfigurationEncryptionSettings(
   };
 
   protected void requireEncryptedFileSystem() {
```

Comment:
how about adding a javadoc saying safe to use as it doesn't need the filesystem

Reply:
sounds good, pushed the update